### PR TITLE
fake odom added

### PR DESCRIPTION
**Pull request by iharsuvorau**
Please, check it out.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# Setup Eigen3 variables
+find_package(Eigen3 REQUIRED)
+
+find_package(catkin REQUIRED COMPONENTS
+  geometry_msgs
+  roscpp
+  rospy
+  std_msgs
+  joy
```
Comment (RValner) [inline CMakeLists.txt]: joy should not be part of the driver package

File: package.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  <license>Apache 2.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_depend>joy</build_depend>
```
Comment (RValner) [inline package.xml]: Remove joy dependency

**Review by RValner — CHANGES_REQUESTED** [review]
Remove dependency from "joy" and this pull request should then be good to go.

**Comment by iharsuvorau** [comment]
@RValner No more joy in d4dcc03.

**Review by RValner — APPROVED** [review]

**Pull request merged**